### PR TITLE
test: [M3-8133] - Add Cypress test for Login History page

### PR DESCRIPTION
**Pull request by cliu-akamai**
## Description 📝
Add integration test for Login History page.

## Major Changes 🔄
- Check account logins are shown in the table.
- Check "No account logins" message shows up when there is no item in the table.

## How to test 🧪
```
yarn cy:run -s "cypress/e2e/core/account/account-login-history.spec.ts"
```

### CODE REVIEW

File: packages/manager/cypress/e2e/core/account/account-login-history.spec.ts [hunk 1]
```diff
@@ -137,13 +137,117 @@ describe('Account login history', () => {
...
+
+      // Confirm that unrestricted user login displays in table.
+      cy.findByText(mockSuccessfulLogin.username)
+        .should('be.visible')
+        .closest('tr')
+        .within(() => {
+          // Confirm that successful login and status icon display in table.
+          // cy.findByText(mockSuccessfulLogin.status, { exact: false }).should(
+          //   'be.visible'
+          // );
```
Comment (dwiley-akamai) [inline packages/manager/cypress/e2e/core/account/account-login-history.spec.ts:211]: Is this (and L196-198) commented out pending something that's forthcoming or can it be removed now?

**Review by dwiley-akamai — APPROVED** [review]
Test passed locally & remotely ✅

Can we add a changeset under the `Tests` category?

**Review by jdamore-linode — APPROVED** [review]
Nice work @cliu-akamai! Approved pending both the points Dajahi mentioned:

- Changeset addition
- Stray comments that could possibly be cleaned up

**Comment by jdamore-linode** [comment]
@cliu-akamai You'll probably have to pull in the latest changes from `develop` to fix those OBJ failures!

**Pull request merged**